src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -20,6 +20,8 @@
     <ShouldVerifyClosure>false</ShouldVerifyClosure>
     <!-- Publish crossgen2 as a single-file app on native-OS builds. Cross-OS NativeAOT compilation is not supported yet -->
     <NativeAotSupported Condition="'$(CrossBuild)' == 'true' and '$(TargetOS)' != '$(HostOS)'">false</NativeAotSupported>
+    <!-- Can't use NativeAOT in source build yet https://github.com/dotnet/runtime/issues/66859 -->
+    <NativeAotSupported Condition="'$(DotNetBuildFromSource)' == 'true'">false</NativeAotSupported>
```

Comment:
This is a simple revert to the previous state. I do not want to be fixing it to avoid introducing more build breaks.

We want to get rid of this line anyway.

Reply:
(This is reverting #97536.)

Reply:
Fair enough